bindings/wallet-js/src/lib.rs

```diff
@@ -68,21 +77,26 @@ pub struct BlockDate(chain_impl_mockchain::block::BlockDate);
 
 #[wasm_bindgen]
 impl Wallet {
-    /// retrieve a wallet from the given mnemonics and password
-    ///
-    /// this function will work for all yoroi, daedalus and other wallets
-    /// as it will try every kind of wallet anyway
+    /// Recovers a wallet from the given BIP39 mnemonics and password.
     ///
     /// You can also use this function to recover a wallet even after you have
-    /// transferred all the funds to the new format (see the _convert_ function)
+    /// transferred all the funds to the new format (see the `convert` method).
     ///
-    /// the mnemonics should be in english
+    /// The mnemonics should be in English, or normalized using NFKD.
```

Comment:
I don't think this is an **or** but an **and**. The character encoding shouldn't have anything to do with the language. We have all the dictionaries, but we are only checking with the english one.